content/en/real_user_monitoring/session_replay/_index.md

```diff
@@ -1,38 +1,46 @@
+  tag: 'Blog'
+  text: 'Use funnel analysis to understand and optimize key user flows'
+- link: '/real_user_monitoring/explorer'
+  tag: 'Documentation'
+  text: 'Visualize your RUM data in the Explorer'
 ---
 
 ## Overview
 
-Session Replay expands your user experience monitoring by allowing you to capture and visually replay the web browsing experience of your users.
+Session Replay expands your user experience monitoring by allowing you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls.
 
-Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls.
+The RUM Browser SDK is [open source][1] and leverages the open source [rrweb][2] project.
 
-## Collect Session Replay data
+## Session Replay Recorder
 
-### Initial setup
+The Session Replay recorder, part of the RUM Browser SDK, takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with their timestamps. 
 
-To use Session Replay, set up [Datadog RUM Browser Monitoring][1]. Set up the following sections: application creation, client token generation, and RUM SDK setup.
+Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 day retention policy as normal RUM sessions. 
 
-Session Replay is available through a dedicated build of the RUM Browser SDK. To enable Session Replay, change the npm package name or CDN URL, depending on your chosen installation method:
+The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3]. 
 
-#### npm
+In order to reduce Session Replay's network impact and ensure the Session Replay recorder has minimal overhead on your application's performance, Datadog compresses the data prior to sending it, and reduces the load on a browser’s UI thread by delegating most of the CPU-intensive work (such as compression) to a background service worker. The expected network bandwidth impact is less than 100Kb/min. 
 
-Replace the `@datadog/browser-rum package` with a version >3.6.0 of [`@datadog/browser-rum`][2]. To start the recording, call `datadogRum.startSessionReplayRecording()`.
+## Setup
+
+Session Replay is available through a dedicated build of the RUM Browser SDK. To start collecting data for Session Replay, set up [Datadog RUM Browser Monitoring][4] by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK.
```

Comment:
```suggestion
Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, set up [Datadog RUM Browser Monitoring][4] by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK.
```
Just checked with @DataDog/rum-browser and this sentence isn't accurate anymore. There is no dedicated build, SR is included in the standard bundle/package now.